sdk/eventhub/azure-eventhub/azure/eventhub/aio/_consumer_async.py

```diff
@@ -165,7 +165,7 @@ def _next_message_in_buffer(self):
         # pylint:disable=protected-access
         message = self._message_buffer.pop()
         event_data = EventData._from_message(message)
-        trace_link_message(event_data)
+        get_event_links(event_data)
```

Comment:
same as the sync one, wondering how `get_event_links` could differentiate from the `trace_link_message`